app/workers/github_repos/repo_sync_worker.rb

```diff
@@ -37,6 +37,12 @@ def perform(repo_id)
              Github::Errors::AccountSuspended,
              Github::Errors::RepositoryUnavailable
         repo.destroy
+      rescue Github::Errors::ClientError => e
+        if e.message.include? "Repository access blocked"
+          repo.destroy
```

Comment:
Is there any reason or value in adding some kind of user facing notification here? I would be disappointed if my configuration were purged because of an error that turned out to be transient - and surprised if my configuration changed in any case.

Reply:
It's possible, but that may be a product decision — I typically try to leave anything user-facing to them.

We should probably have _some_ way of resynching GitHub repos, though. I don't know if we pick up new repos created after the original fetch.

Reply:
I've merged this to reduce the noise for the moment, but I'll open a stage-0 RFC to discuss notifying users when we remove their GitHub repos.